src/analyses/variable-sensitivity/variable_sensitivity_object_factory.cpp

```diff
@@ -126,7 +127,7 @@ abstract_object_pointert variable_sensitivity_object_factoryt::
     return initialize_abstract_object<abstract_objectt>(
       followed_type, top, bottom, e, environment, ns);
   default:
-    assert(false);
+    UNREACHABLE;
```

Comment:
This was a bit of drive-by cleanup to keep the Linter happy...

Reply:
Cleanups should go into a separate commit.